doc/refman/RefMan-pre.tex

```diff
@@ -1087,6 +1087,140 @@ \section*{Credits: version 8.5}
+fixing some incompatibilities introduced in Coq 8.5. Unification
+constraints can now be left floating around and be seen by the user
+thanks to a new option. The {\tt Keyed Unification} mode has been
+improved by Matthieu Sozeau. 
+
+The typeclass resolution engine and associated proof-search tactic have
+been reimplemented on top of the proof-engine monad, providing better
+integration in tactics, and new options have been introduced to control
+it, by Matthieu Sozeau with help from Théo Zimmermann.
+
+The efficiency of the whole system has been significantly improved
+thanks to contributions from Pierre-Marie Pédrot, Maxime Dénès and
+Matthieu Sozeau and performance issue tracking by Jason Gross and Paul
+Steckler.
+
+Standard library improvements by Jason Gross, Sébastien Hinderer, Pierre
+Letouzey and others.
+
+Emilio Jesús Gallego Arias contributed many cleanups and refactorings of
+the pretty-printing and user interface communication components.
+
+Frédéric Besson maintained the micromega tactic.
+
+The OPAM repository for {\Coq} packages has been maintained by Guillaume
+Claret, Guillaume Melquiond, Matthieu Sozeau, Enrico Tassi and others. A
+list of packages is now available at \url{https://coq.inria.fr/opam/www/}.
+
+Packaging tools and software development kits were prepared by Michael
+Soegtrop with the help of Maxime Dénès and Enrico Tassi for Windows, and
+Maxime Dénès and Matthieu Sozeau for MacOS X. Packages are now regularly
+built on the continuous integration server.
+
+Matej Košík maintained and greatly improved the continuous integration
+setup and the testing of {\Coq} contributions. He also contributed many
+API improvement and code cleanups throughout the system.
+
+The contributors for this version are C.J. Bell, Yves Bertot, Frédéric
+Besson, Tej Chajed, Pierre Courtieu, Maxime Dénès, Ricky Elrod, Jason
+Gross, Hugo Herbelin, Emilio Jesus Gallego Arias, Jacques-Henri Jourdan,
+Matej Košík, Xavier Leroy, Pierre Letouzey, Gregory Malecha, Cyprien
+Mangin, Erik Martin-Dorel, Guillaume Melquiond, Pierre-Marie Pédrot,
+Lionel Rieg, Gabriel Scherer, Matthieu Sozeau, Arnaud Spiwack, Paul
+Steckler, Laurent Théry, Enrico Tassi, Nickolai Zeldovich, Théo
+Zimmermann and Daniel de Rauglaudre. The development process was
+coordinated by Hugo Herbelin and Matthieu Sozeau with the help of
+Maxime Dénès, who was also in charge of the release process.
+
+Many power users helped to improve the design of the new features via
+the bug tracker, the pull request system, the {\Coq} development mailing
+list or the coq-club mailing list. Special thanks to the users who
+contributed patches and intensive brain-storming and code reviews,
+starting with Cyril Cohen, Jason Gross, Robbert Krebbers, Jonathan
+Leivent, Xavier Leroy, Gregory Malecha, Clément Pit-Claudel, Gabriel
+Sherer and Beta Ziliani. It would however be impossible to mention
+exhaustively the names of everybody who to some extent influenced the
```

Comment:
"Scherer". But maybe a synthesis should be done with the "The contributors ..." paragraph to avoid redundancies. 

Strictly speaking, Clément has also a contribution.